language/fr/buildings.lang

```diff
@@ -1,5 +1,185 @@
+$_Lang['InfoBox_DestroyCost']                   = 'Réduire le coût';
+$_Lang['InfoBox_DestroyTime']                   = 'Réduire le temps';
+
+$_Lang['ListBox_StructuresList']                = 'Liste des bâtiments';
+$_Lang['ListBox_Disallow_LabResearch']          = 'Recherche en cours';
+$_Lang['ListBox_Disallow_NoTech']               = 'Prérequis manquant';
+$_Lang['ListBox_Disallow_MaxLevelReached']      = 'Niveau maximal atteint !';
+$_Lang['ListBox_Disallow_NoFreeFields']         = 'Aucune case disponible !';
+$_Lang['ListBox_Disallow_QueueIsFull']          = 'La file d\'attente est pleine !';
+$_Lang['ListBox_Disallow_NoResources']          = 'Ressources insuffisantes !';
+$_Lang['ListBox_Disallow_NoElements']           = 'Rien à démolir !';
+$_Lang['ListBox_Disallow_VacationMode']         = 'Mode vacances';
+
+$_Lang['Queue_Header']                          = 'Constructions en cours';
+$_Lang['Queue_Empty']                           = 'Aucune';
+$_Lang['Queue_Full']                            = 'Liste pleine !';
+$_Lang['Queue_EndTime']                         = 'Terminé ';
+$_Lang['Queue_Mode_Destroy_1+']                 = 'Démolir';
+$_Lang['Queue_Mode_Build_1+']                   = 'Construire';
+$_Lang['Queue_Mode_Destroy_1']                  = 'en démolition';
+$_Lang['Queue_Mode_Build_1']                    = 'en construction';
+$_Lang['Queue_Mode_Research_1']                 = 'en recherche';
+$_Lang['Queue_CantCancel_Premium']              = 'Tu ne peux pas annuler une construction lancée avec de l‘énergie noire !';
+$_Lang['Queue_ConfirmCancel']                   = 'Es-tu sûr de vouloir annuler cette construction ?';
+$_Lang['Queue_Cancel_CantCancel']               = 'Ne peut pas être annulé';
+$_Lang['Queue_Cancel_Destroy']                  = 'Annuler';
+$_Lang['Queue_Cancel_Build']                    = 'Annuler';
+$_Lang['Queue_Cancel_Remove']                   = 'Retirer de la liste';
+$_Lang['Queue_Cancel_Go']                       = 'Continuer';
+$_Lang['Queue_EndTitleBeg']                     = 'Terminé le ';
+$_Lang['Queue_EndTitleHour']                    = 'à';
+
+// Laboratory
+$_Lang['Research']                              = 'Centre de recherche';
+
+$_Lang['ListBox_TechsList']                     = 'Liste des technologies';
+$_Lang['ListBox_Disallow_NotThisLab']           = 'Recherche lancée sur une autre planète !';
+$_Lang['ListBox_Disallow_LabInQueue']           = 'Centre de recherche en cours de construction !';
+$_Lang['ListBox_Disallow_NoLab']                = 'Centre de recherche manquant !';
+
+$_Lang['InfoBox_SelectTech']                    = 'Choisis une technologie';
+$_Lang['InfoBox_DoResearch']                    = 'Améliorer<br/>Niveau';
+$_Lang['InfoBox_ResearchTime']                  = 'Durée ';
+
+$_Lang['Queue_ResearchOn']                      = 'Recherche en cours sur ';
+$_Lang['Queue_CantCancel_Premium_Lab']          = 'Tu ne peux pas annuler une construction lancée avec de l‘énergie noire !';
```

Comment:
This, and the next line mention "construction". Shouldn't this be changed to an equivalent of english "research"?

Reply:
From your comment, I assume these 2 strings are only used for the research lab and not for the shipyard... If so, OK for the change. Done.